src/Microsoft.TestPlatform.Client/AttachmentsProcessing/InProcessTestRunAttachmentsProcessingEventsHandler.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+
+using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel.Client;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel.Logging;
+
+namespace Microsoft.VisualStudio.TestPlatform.Client;
+
+internal class InProcessTestRunAttachmentsProcessingEventsHandler : ITestRunAttachmentsProcessingEventsHandler
+{
+    private readonly ITestRunAttachmentsProcessingEventsHandler _oldEventsHandler;
+
+    public InProcessTestRunAttachmentsProcessingEventsHandler(
+        ITestRunAttachmentsProcessingEventsHandler oldEventsHandler)
+    {
+        _oldEventsHandler = oldEventsHandler;
+    }
+
+    public void HandleLogMessage(TestMessageLevel level, string? message)
+    {
+        _oldEventsHandler.HandleLogMessage(level, message);
+    }
+
+    public void HandleProcessedAttachmentsChunk(IEnumerable<AttachmentSet> attachments)
+    {
+        // Not implemented by design, keep in sync with the same named method from
+        // TestRunAttachmentsProcessingEventsHandler.cs.
+        throw new NotImplementedException();
```

Comment:
I tend to prefer having some explanation in such kind of exception. If we are really "sure" this is not supposed to be called, I would add some `Debug.Fail` call in front so that we could see failures in our tests if we break this "contract".

Reply:
The comment is no longer relevant since I removed the wrapper class altogether, but answering the question for posterity: we are indeed sure this is not supposed to be called, because as far as I can tell, the original attachment processing logic doesn't use this method at all. It may have been added to be used later, but it's not been the case to date.